passkey-types/src/webauthn/attestation.rs

```diff
@@ -627,42 +914,178 @@ mod tests {
     }
 
     #[test]
-    fn webauthn_me_debugger() {
-        let request = r#"{
-            "publicKey": {
-              "rp": {
-                "name": "test"
-              },
-              "user": {
-                "id": [
-                  208, 3, 44, 155, 74, 109, 149, 31, 234, 107, 36, 243, 249, 29, 32, 48,
-                  189, 69, 220, 216, 11, 222, 113, 155, 129, 208, 156, 217, 58, 99, 41,
-                  166
-                ],
-                "name": "test",
-                "displayName": "Test User"
-              },
-              "challenge": [
-                21, 69, 217, 214, 15, 130, 240, 139, 91, 76, 136, 60, 96, 131, 25, 110,
-                173, 121, 215, 220, 246, 162, 39, 30, 0, 144, 238, 65, 195, 219, 32, 233
-              ],
-              "pubKeyCredParams": [
-                {
-                  "type": "public-key",
-                  "alg": "-257"
```

Comment:
**Issue:** A previous test seems to have been overwritten. Perhaps a bad git conflict resolution?

Reply:
ah yes this is very bad... will fix